js/digifinex.js

```diff
@@ -427,6 +436,7 @@ module.exports = class digifinex extends Exchange {
                 };
             }
             result[code]['networks'][networkId] = network;
```

Comment:
it's because the `chain` field has empty string value in raw api response: https://imgsh.net/a/INHc9jF.png
some other entries do that value in that field, some not. for those, it's returned to `undefined`.
do you have a suggestion what should we do you in that case? `'unknown'` or  `'undefined'`  or just emptry string `''` as key?

Reply:
@ttodua but isn't that a (weird) way of saying that's the native chain? If so we could make networkId = CODe

Reply:
i am not sure whether we should set `code` to `networkId`, as it's "network id" literally, we should leave it as exchange requires i.e. if it requires that while doing fetchDepositAddress or whatever, the `network` value in request might be required to be same empty string as provided in currencies>network response.
 i.e. if it says `''` (empty string) as BTC's chain id, we might be required to provide that empty string in fDA request (or other requests related to network-dependent actions), and if we artificially set the 'id' to whatever value (doesnt matter if that's CODE or some title) as we might be making fault in that case. 
wdyt?

Reply:
![image](https://user-images.githubusercontent.com/43336371/205946662-2a7e67f2-d547-440b-aa24-93d6a15c5bfc.png)
![image](https://user-images.githubusercontent.com/43336371/205946759-575d4e4e-a6b2-4965-94d1-389d630464c0.png)

⬆️ Apparently we only need the chain for USDT, so that won't be a problem, I just don't know what we should use as the network name, undefined or empty string seems a bit weird to me but the CODE might not be correct either because it might be an ERC20 token or smth

Reply:
well, what should we do now? unless you tell me which one, i woudln't submit per my discretion (even though empty string might be my chosen one), so let me know your decision (btw, might not need much brainstorming for this low-end exchange, whichever seems more future-proof way, lets do that way).

Reply:
@ttodua actually, we are overcomplicating it here; let's not add the network if chain == "", I think that fixes it

Reply:
the implementation was not build by me, so I thought it's better not to change that, for 2 reasons:
1) it will be breaking change
2) beside breaking change (to ignore that) , the implementation will be regress (as we are moving to unification, this will be regress).

so, i still think that it's quite good that we had networks, and there is nothing that prevents us from doing that, the matter we discuss is a very minor obstacle and it's not worth make regress because of that small obstacle.

Reply:
I don't see any downside in omitting this "empty network" that nobody really knows what it is

Reply:
> I don't see any downside in omitting this "empty network" that nobody really knows what it is

But won't that shoot our feet if we make non-standard response for same exchange? some objects having networks inside and some not? I feel that it's not correct way imo... maybe instead of empty string we can just set `'default'` string, or `'undetected | undetermined'`  or whatever, we should think up such cases. I don't see downside in this. 

or instead of `networks` we might add another key `defaultNetwork`... 

well, but as you decide, just my opinions.

Reply:
> But won't that shoot our feet if we make non-standard response for same exchange?

@ttodua here are not the ones making it non-standard, having an "" network makes it very non-standard so not oyr fault and as you said we should not spend too much time with this exchange, so the simpler fix here the best

Reply:
the thing was that digifinex fetchCurrencies was not implemented by me, and the way it's implemented, requires more coding and rework to achieve that variable structure, but sure, I'll do that now.

Reply:
@ttodua thanks

Reply:
I had submitted the fix that day, sorry for not letting you know explicitly.

Reply:
@ttodua no problem, will take a look